<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab</class>
 <widget class="QFrame" name="Tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QFrame" name="displayFrame">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>900</width>
     <height>900</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="codeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>961</width>
     <height>871</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QTabWidget" name="terminalWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>930</y>
     <width>1880</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
